pkg/query/api/v1.go

```diff
@@ -294,7 +310,7 @@ func (api *API) query(r *http.Request) (interface{}, []error, *ApiError) {
 	defer span.Finish()
 
 	begin := api.now()
-	qry, err := api.queryEngine.NewInstantQuery(api.queryableCreate(enableDedup, 0, enablePartialResponse, warningReporter), r.FormValue("query"), ts)
+	qry, err := api.queryEngine.NewInstantQuery(api.queryableCreate(enableDedup, replicaLabels, 0, enablePartialResponse, warningReporter), r.FormValue("query"), ts)
```

Comment:
We are bit inconsistent here again. 

Everywhere else default values are returned from `parseXXXParam` if param is empty. Can we do the same for `parseReplicaLabelsParam`?

(current version gets default replica label from `api.queryEngine`

Reply:
updated